apis/core/v1beta1/openstackcontrolplane_webhook.go

```diff
@@ -902,3 +937,27 @@ func (r *OpenStackControlPlane) DefaultLabel() {
 		r.Labels[typeLabel] = ""
 	}
 }
+
+// ValidateTLSData checks if the TLS data in the apiOverride are complete
+func validateTLSOverrideSpec(override **route.OverrideSpec, basePath *field.Path) field.ErrorList {
+	var allErrs field.ErrorList
+
+	if *override == nil {
+		return allErrs
+	}
+
+	tlsSpec := (*override).Spec
+	if tlsSpec != nil && tlsSpec.TLS != nil {
+		if tlsSpec.TLS.CACertificate == "" {
```

Comment:
CA cert is optional IIRC

Reply:
You mean when use the hostname validation using service name? I didn't try this specific case, but when we don't include the CA cert, `destinationCACertificate` is missing error shows up, and route is not created.

Reply:
Yes, it's optional, when the CA cert is not used, the bundle one is